Crypto/Crypto/Presentation/Views/HomeView/HomeViewModel.swift

```diff
@@ -12,8 +12,13 @@ class HomeViewModel: ObservableObject{
     @Environment(\.colorScheme) var colorScheme
     @Published var isDarkMode = false
     
-    func toggleDarkMode(){
-        self.isDarkMode.toggle()
+    func toggleMode(mode: String){
+        if(mode == "ligthMode"){
```

Comment:
Overall it looks good! Even though there are only two options (dark and light mode) I would suggest handling this with an Enum and a Switch/Case statement, as it is less prone to errors and easier to understand in the code if the states become more complex.